docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md

```diff
@@ -121,37 +121,58 @@ for the `geoLocation` of the `WindTurbine` and `WindFarm`.
 
 ## Updating the Spreadsheet.
 
-Olav copies over all the classes and properties from the enterprise model to the solution model. He then removes
-the rows that are not needed for the new solution model. The spreadsheet will now look as follows
-(details of other columns excluded on purpose):
-
-| Class       | Property       | Value Type  |
-|-------------|----------------|-------------|
-| WindTurbine | name           | string      |
-| WindTurbine | type           | string      |
-| WindTurbine | activePower    | timeseries  |
-| WindTurbine | geoLocation    | Point       |
-| WindTurbine | manufacturer   | string      |
-| WindTurbine | ratedPower     | float       |
-| WindTurbine | hubHeight      | float       |
-| WindTurbine | lifeExpectancy | integer     |
-|             |                |             |
-| WindFarm    | name           | string      |
-| WindFarm    | geoLocation    | Polygon     |
-| WindFarm    | windTurbines   | WindTurbine |
-|             |                |             |
-| GeoLocation | name           | string      |
-|             |                |             |
-| Point       | latitude       | float       |
-| Point       | longitude      | float       |
-|             |                |             |
-| Polygon     | point          | Point       |
+Olav copies over all the properties from the enterprise model to the solution model. He then removes
+the rows that are not needed for the new solution model. The `Properties` spreadsheet will now look as follows
+(only the most relevant columns shown here):
+
+| Class       | Property       | Value Type  | ... | Reference                                  |
+|-------------|----------------|-------------|-----|--------------------------------------------|
+| WindTurbine | name           | string      |     | power:GeneratingUnit(property=name)        |
+| WindTurbine | type           | string      |     | power:GeneratingUnit(property=type)        |
+| WindTurbine | activePower    | timeseries  |     | power:GeneratingUnit(property=activePower) |
+| WindTurbine | geoLocation    | Point       |     | power:GeneratingUnit(property=geoLocation) |
+| WindTurbine | manufacturer   | string      |     | power:WindTurbine(property=manufacturer)   |
+| WindTurbine | ratedPower     | float       |     | power:WindTurbine(property=ratedPower)     |
+| WindTurbine | hubHeight      | float       |     | power:WindTurbine(property=hubHeight)      |
+| WindTurbine | lifeExpectancy | integer     |     | power:WindTurbine(property=lifeExpectancy) |
```

Comment:
this is so powerful ! really lego style modeling :)

Reply:
just wandering since we are not explicitly stating version of view, do we need this to be mentioned?